prow/plugins/milestonestatus/milestonestatus.go

```diff
@@ -46,7 +48,20 @@ type githubClient interface {
 }
 
 func init() {
-	plugins.RegisterGenericCommentHandler(pluginName, handleGenericComment, nil)
+	plugins.RegisterGenericCommentHandler(pluginName, handleGenericComment, helpProvider)
+}
+
+func helpProvider(config *plugins.Configuration, enabledRepos []string) (*pluginhelp.PluginHelp, error) {
+	return &pluginhelp.PluginHelp{
+			Description: "The milestonestatus plugin allows members of the milestone maintainers github team to specify the 'status/*' label that should apply to a pull request.",
+			WhoCanUse:   "Members of the milestone maintainers github team can use the '/status' command. This team is specified in the config by providing the github team's ID.",
+			Usage:       "/status [approved-for-milestone|in-progress|in-review]",
+			Examples:    []string{"/status approved-for-milestone", "/status in-progress", "/status in-review"},
+			Config: map[string]string{
+				"": fmt.Sprintf("The milestone maintainers team is the github team with ID: %d.", config.MilestoneStatus.MaintainersID),
```

Comment:
Can we make a link here to the actual team instead of providing the ID?

Reply:
You can't create a link from a team ID as far as I'm aware. We could provide a link (and the team name), but it would require an additional API token every time help is fetched from hook.